packages/frontend/app/models/channel.ts

```diff
@@ -1,7 +1,9 @@
 import Model, { attr, hasMany } from '@ember-data/model';
 
 import Identity from 'emberclear/models/identity';
-import Invitation from 'emberclear/models/invitation';
+import ChannelContextChain from './channel-context-chain';
+import VoteChain from './vote-chain';
+// import Invitation from 'emberclear/models/invitation';
```

Comment:
These comments should be removed unless there is a plan to bring them back?

Reply:
Sounds good. Will do that around 2PM.